### PR TITLE
allow swearing in #filter-requests

### PR DESCRIPTION
**Pull request by tarna**
allow swearing in #filter-requests on the Minehut Meta Discord

### CODE REVIEW

File: src/guild/config/guildConfigs.ts [hunk 1]
```diff
@@ -197,9 +197,18 @@ guildConfigs.set('546414872196415501', {
 		},
 		censor: {
 			inviteWhitelist: INVITE_WHITELIST,
-			allowSwearing: false,
```
Comment (drtshock) [inline src/guild/config/guildConfigs.ts:200]: Is this false by default since you removed it?

Comment (drtshock) [inline src/guild/config/guildConfigs.ts:200]: Is this false by default since you removed it entirely?

Comment (jellz) [reply src/guild/config/guildConfigs.ts:200]: Yeah, swearing will be disallowed by default if `allowSwearing` is omitted

**Comment by BennyDoesStuff** [comment]
This channel no longer exists, closing.

**Pull request closed without merging**